src/apps/competitions/unpackers/codabench_v1.py

```diff
@@ -0,0 +1,211 @@
+import os
+
+from django.utils import timezone
+
+from competitions.unpackers.base_unpacker import BaseUnpacker
+from .utils import CompetitionUnpackingException
+
+
+class CodabenchV1Unpacker(BaseUnpacker):
```

Comment:
this seems to have quite a few DRY violations?

Reply:
Hi Eric, From your opinion,  a better approach would be to inherit V2Unpacker and then add the features I need on top of that, right?

Reply:
Hi Eric, i make a changed already, can u review it, thanks.